privacy_partner_report/controllers/main.py

```diff
@@ -0,0 +1,81 @@
+# Copyright 2018 Eficent Business and IT Consulting Services, S.L.
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl).
+
+import base64
+import json
+import time
+import zlib
+from openerp import http
+from openerp.http import request, content_disposition
+from openerp.service.report import exp_report, exp_report_get
+
+from openerp.addons.web.controllers.main import serialize_exception
+from openerp.addons.web.controllers.main import Reports
+
+
+class ReportsExtended(Reports):
+
+    def _update_report_datas_or_data(self, action, report_ids, report_data):
+        if 'datas' in action:
+            if 'ids' in action['datas']:
+                report_ids = action['datas'].pop('ids')
+            report_data.update(action['datas'])
+        elif 'data' in action:
+            if 'ids' in action['data']:
+                report_ids = action['data'].pop('ids')
+            report_data.update(action['data'])
+        return report_ids, report_data
+
+    @http.route('/web/report', type='http', auth="user")
```

Comment:
You can just use `@http.route()` if the other parameters didn't change.

Reply:
I know